db/migrate/20201029093313_add_user_type_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserTypeToVersions < ActiveRecord::Migration[6.0]
+  def change
+    add_column :versions, :user_type, :string
```

Comment:
rather than user type, maybe this could be `supplier_id` ? nil for a FE user, otherwise the id of the supplier for a BE user

Reply:
If I were to change this to `supplier_id`, would `user_for_paper_trail` want changing to return `X-Current-User or nil`? Currently, it's returning the supplier id

Reply:
oh sorry, I didn't spot that. We shouldn't mix strings with uuids - I think `user_for_paper_trail` should be a string, and should be the `headers['X-Current-User']` or nil.

We may want to encourage suppliers to also send the user in the future, so this field represent either an MoJ user or a supplier user.